experimental/packer/configure.sh

```diff
@@ -130,8 +130,6 @@ gem install bundler
 
 #deb-s3 upload --bucket my-bucket my-deb-package-1.0.0_amd64.deb
 
-docker run -d -p 5000:5000 --name registry registry:2
```

Comment:
That related to the (deprecated) mechanism for running jobs within rush as docker containers.